src/transformers/models/auto/processing_auto.py

```diff
@@ -43,6 +43,7 @@
     [
         ("altclip", "AltCLIPProcessor"),
         ("blip", "BLIPProcessor"),
+        ("blip-2", "BLIP2Processor"),
```

Comment:
There is a typo here.
```suggestion
        ("blip-2", "Blip2Processor"),
```